packages/cdktf-cli/lib/get/generator/provider-schema.ts

```diff
@@ -43,7 +100,7 @@ export type AttributeType =
   | ["list", AttributeType]
   | ["object", { [attribute: string]: AttributeType }];
 
-export type BlockType =
+export type BlockType =  // FIXME: use such a type for NestedType Attributes as well?
```

Comment:
Might not hurt, but seems to be over specifying compared to the proto definition.

Reply:
yeah, right.